Sources/BothamViewController.swift

```diff
@@ -11,31 +11,34 @@ import Foundation
     import UIKit
 
 public class BothamViewController: UIViewController, BothamUI {
-    public var presenter: BothamPresenter! = nil
+    private var _presenter: BothamPresenter? = nil
 
     public override func viewDidLoad() {
         super.viewDidLoad()
-        presenter.viewDidLoad()
+        if self.respondsToSelector(Selector("presenter")) {
+            _presenter = self.performSelector(Selector("presenter")).takeUnretainedValue() as? BothamPresenter
+        }
+        _presenter?.viewDidLoad()
```

Comment:
We throw the exception to avoid null checks, but we can just print a log message in this implementation. Up to you :)